docs/src/main/tut/generate-sources-from-idl/compendium.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: docs
+title: Generating sources from IDLs stored in Compendium
+permalink: /generate-sources-from-compendium
+---
+
+# Generating sources from IDLs stored in [Compendium](https://higherkindness.io/compendium/)
+
+compendium is a standalone solution, implemented as an HTTP service, that provides storage, conversion and client generation for your schemas in a format-agnostic fashion. `sbt-mu-srcgen` provides a feature that enables the user to store and get IDL definitions from compendium.
+
+This section will provide instructions about how to configure the sbt settings to use compendium. We are assuming you have a compendium instance running. If not, please check the [compendium microsite](https://higherkindness.io/compendium/).
+Also, the configuration related to the IDL type could be checked on:
+
+* [Protobuf section](generate-sources-from-proto)
+* [Avro section](generate-sources-from-avro))
+* [OpenAPI section](generate-sources-from-openapi))
+
+## Configure sbt plugin
+
+Settings related to compendium interaction are:
+
+| Setting                                 | Type                      | Description                                                                                                                                                                                             | Default value            |
+|:----------------------------------------|:--------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------------|
+| `muSrcGenExecutionMode`                 | ExecutionMode             | Execution mode of the plugin. If `Compendium, it's required a compendium instance where IDL files are saved. | `Local`                                                                                  |                          |
+| `muSrcGenCompendiumServerUrl`           | _String_                  | Compendium server url                                                                                                                                                                                   | `http://localhost:47047` |
+| `muSrcGenCompendiumProtocolIdentifiers` | _Seq[ProtocolAndVersion]_ | Protocol identifiers to retrieve from compendium. `ProtocolAndVersion` provides two values: `name` (mandatory) that corresponds with the identifier used to store the protocol and `version` (optional) | `Nil`                    |
+
```

Comment:
I'm not sure but... maybe we have to include too `muSrcGenSerializationType `??
As we can see [here](https://github.com/higherkindness/sbt-mu-srcgen/blob/335add7b93a3e2e999429a45086e426d10968f42/plugin/src/main/scala/higherkindness/mu/rpc/srcgen/SrcGenPlugin.scala#L49-L54) it's `Avro` by default but it would be changed if we want to work with others IDLs, what do you think?

Reply:
In order to not duplicate content, I referenced the configuration related to the IDL files. But maybe it's not very intuitive.. 🤔